### PR TITLE
fixed error handling

### PR DESCRIPTION
**Pull request by revomhere**

### CODE REVIEW

File: packages/fetcher/src/response-builder.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class FetcherResponseBuilder<T> {
   }
 
   async #extractData() {
-    if (!this.#response || !this.#response.ok) return
+    if (!this.#response) return
```
Comment (napalmpapalam) [inline packages/fetcher/src/response-builder.ts:44]: cover with tests

File: packages/jac/src/errors/server-errors.ts [hunk 2]
```diff
@@ -65,6 +65,13 @@ export class JsonApiError extends Error {
   get nestedErrors(): JsonApiResponseNestedErrors {
     return this.#nestedErrors
   }
+
+  /*
+   * Original error
+   */
+  get originalError() {
```
Comment (napalmpapalam) [inline packages/jac/src/errors/server-errors.ts]: add return type

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0-rc.13] - 2023-05-18
+### Fixed
+- `@distributedlab/fetcher` - fixed error building in response
```
Comment (napalmpapalam) [inline CHANGELOG.md]: avoid using "fixed", "added", or "changed" words in the specific section of the change log coz the section is self-descriptive - it's obvious that u fixed something if u add change under `###Fixed` heading

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

File: packages/fetcher/src/response-builder.test.ts [hunk 4]
```diff
@@ -45,4 +50,36 @@ describe('performs FetcherResponseBuilder', () => {
 
     expect(result.data).toEqual(blob)
   })
+
+  test('should build valid error response', async () => {
+    const response = new Response(
+      '{"errors":[{"id":"1","code":"err_some_code","status":"401","title":"Unauthorized"}]}',
```
Comment (lukachi) [inline packages/fetcher/src/response-builder.test.ts]: move into const and use JSON.stringify here

File: packages/w3p/src/providers/wrapped/fallback-evm.ts [hunk 5]
```diff
@@ -31,7 +31,7 @@ export class FallbackEvmProvider
     this.#provider = provider as unknown as providers.Web3Provider
   }
 
-  static get providerType(): PROVIDERS {
+  static get providerType(): string {
```
Comment (ArtSuslov) [inline packages/w3p/src/providers/wrapped/fallback-evm.ts:34]: `PROVIDERS[keyof PROVIDERS]`

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

**Pull request merged**